services/PA-accounts-service/src/data-storage/data-storage.controller.ts

```diff
@@ -44,7 +44,9 @@ export class DataStorageController {
   public async get(
     @User('id') userId: number,
     @Param() params,
-  ): Promise<string> {
-    return await this.dataStorageService.get(userId, params.type);
+  ): Promise<{ data: string | number | object }> {
+    const data = await this.dataStorageService.get(userId, params.type);
+
+    return { data };
```

Comment:
This ensures us that the endpoint always returns JSON.

But the contents of the `data`-property is still a string. (A stringified-JS-object/string).